_component_design/tables-basic.md

```diff
@@ -0,0 +1,156 @@
+---
+title: Basic Tables
+parent: Tables
+layout: component
+category: Components
+usage: >
+  Tables present data on nearly every page in Helix control panels. Tables are highly valuable, but always run the risk of putting too high a cognitive load on users. Our goal with tables should always be to decrease cognitive load so users can get the data they need quickly and easily.
+preview-image: preview-images/tables.svg
+status: Stable
+resource: true
+last-modified: 2017-10-24
+---
+
+{% include toc.html %}
+
+## Introduction
+
+<div class="hxRow">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### When to use
+
+Tables are the bread-and-butter component for displaying complex objects in a neat and tidy format. By presenting complex objects in an easy-to-consume format, Helix control panels allow for advanced users to quickly interact with a large amount of data while ensuring the data are presented efficiently in a user-friendly manner for novice users.
+
+### Best practice
+
+- Keep the column count to under seven. 
+- Keep visual clutter to a minimum to maintain a user's focus.
+- When organizing content in a table, focus on the user's primary task.
+- Preserve hierarchies from left to right. (Most important to least important)
+- Provide filtering and sorting table controls to allow users to interact with and select the most important information.
+
```

Comment:
"Tables are best used when the column count is kept to less than seven and visual clutter is kept to a minimum. When organizing content in a table, focus on the user's primary task by preserving hierarchies from left to right (most important to least important), and providing filtering and sorting of table controls to allow users to interact with and select the most important information."

Reply:
Re: We have converted some bullet point lists into paragraphs in the past and I like this approach. After going through 12 or so reviews in the past few days I think we have decided to keep the bullet point lists for the `Best practice` section within each doc. Now reads as below, Note: the bullet points for this section were edited in a previous commit 8441897 to address language concerns.
```
- Keep the column count to under seven. 
- Keep visual clutter to a minimum to maintain a user's focus.
- When organizing content in a table, focus on the user's primary task.
- Preserve hierarchies from left to right. (Most important to least important)
- Provide filtering and sorting table controls to allow users to interact with and select the most important information.
```